internal/services/cdn/cdn_endpoint_custom_domain_resource.go

```diff
@@ -113,6 +114,7 @@ func resourceArmCdnEndpointCustomDomain() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Optional: true,
 							ValidateFunc: validation.StringInSlice([]string{
+								string(cdn.MinimumTLSVersionNone),
```

Comment:
same here - the type `None` should be implied by the property being omitted, so we don't need to add this constant value and can instead default this in the Create/Update and Read?